### DISCUSSION TITLE
nullable Minor Unit Symbol

### DISCUSSION THREAD

**Discussion by tom-NI** [Daniel-Ioannou/flutter_currency_picker#93] [Ideas]
Hi, 
Id love to see the symbol for minor units, (where applicable) added to Currency, as I have a current use case for it. 
For US and Australian Dollars (for exampel) this would be '¢' (U+00A2) for cents, or ‘p’ for pence in the UK?

**Comment by Daniel-Ioannou** [comment]
I agree that adding support for symbols for minor units would be a valuable addition to the package.

Unfortunately, I don't have the time at this period and won't be able to address this immediately. I will implement it when time permits. However, if you're interested and would like to contribute, please feel free to fork the repository, make the necessary changes, and submit a pull request. I'd be more than happy to review and merge any pull requests, whether from you or others, for this feature.

Thank you for your understanding and potential contribution.
